### PR TITLE
Updated documentation.

### PR DESCRIPTION
**Pull request by ashifkhn**
Date: `26th Dec 2023`

Developer Name: `Ashif`

----

## Issue Ticket Number

-   Add the relevant issue ticket number here (e.g., Closes #<issue_number>).

## Description

Provide a concise description of the changes made in this PR.

**Documentation Updated?**

-   [x] Yes
-   [ ] No

_If your feature adds a new API, then documentation related to the feature like API-Contracts, and Data-models  must be updated._

**Breaking Changes**

-   [ ] Yes
-   [x] No

_If your feature introduces breaking changes or if something is missing, please mention the related issue tickets._

**Development Tested?**

-   [ ] Yes
-   [x] No

_Confirm whether the changes have been tested locally during development._

**Tested in Staging?**

-   [ ] Yes
-   [x] No

_Indicate whether the changes have been tested in the staging environment._

**Under Feature Flag**

-   [ ] Yes
-   [x] No

_Specify if the changes are currently under a feature flag._

**Database Changes**

-   [ ] Yes
-   [x] No

_Indicate whether the changes include modifications to the database._

### Screenshots

Attach any relevant screenshots, such as test coverage reports, before and after images, or other visual aids.

## Test Coverage

Provide information on the test coverage for the changes made in this PR. Include details on the types of tests conducted and their outcomes.

## Additional Notes

Include any additional notes, considerations, or explanations that might be helpful for reviewers.

### CODE REVIEW

File: README.md
Comment (vikhyat187) [inline README.md:51]: This runs only after having the docker compose command right, so add that too

File: README.md
Comment (vikhyat187) [inline README.md:67]: Can you write this for the first time step

File: README.md
Comment (vikhyat187) [inline README.md:74]: Steps after the database is connected to the service

File: docker-compose.yml
Comment (vikhyat187) [inline docker-compose.yml:1]: This file is part of another comment you can remove it

**Review by vikhyat187 — CHANGES_REQUESTED** [review]

File: README.md
Comment (Ajeyakrishna-k) [inline README.md:44]:
```suggestion
- Docker Desktop: [Installation steps](https://docs.docker.com/engine/install/)
- MySQL Docker Image:  [Installation steps](https://hub.docker.com/_/mysql)
```

File: README.md
Comment (Ajeyakrishna-k) [inline README.md:41]: Can you add links to installation?
- Do we need JDK or is JRE enough? 
- Different vendors provide their own implementation of JDK, which one to install?

File: README.md
Comment (Ajeyakrishna-k) [inline README.md:38]: These installation steps can be moved to [RDS docs](https://github.com/Real-Dev-Squad/docs) as these can be used for different apps in future.

File: README.md
Comment (Ajeyakrishna-k) [inline README.md:63]: Can you add context for these images?

File: README.md
Comment (Ajeyakrishna-k) [inline README.md:51]: I think we should add an in-memory DB for local development and set up of local my-sql can be optional.
- What are you thoughts @vikhyat187

File: README.md
Comment (Ajeyakrishna-k) [inline README.md:42]: - Ultimate Edition is not free. I think we should avoid adding it here. 
```suggestion
- Any text editor of your choice.
```

**Review by Ajeyakrishna-k — CHANGES_REQUESTED** [review]

File: README.md
Comment (vikhyat187) [reply README.md:51]: It will be good to have the mysql database, as people will have the idea and they can get more chance to learn about relational database.

File: README.md
Comment (vikhyat187) [reply README.md:42]: +1

Comment (ashifkhn) [reply README.md:42]: I had never used IntelliJ Community previously, still found this easy to set up the project. I think we should tell the preferred one here.

File: README.md
Comment (ashifkhn) [reply README.md:38]: I'm not sure what steps needs to be followed for this.

File: README.md
Comment (ashifkhn) [reply README.md:51]: Please elaboreate.

Comment (iamitprakash) [reply README.md:51]: @Ajeyakrishna-k  and @vikhyat187  is this docker image form public repo will generate schema for skill tree, if not then why this is need... CC- @ashifkhn

Comment (vikhyat187) [reply README.md:51]: yes @iamitprakash  the docker image we are using is a public image.

File: README.md
Comment (vikhyat187) [reply README.md:38]: I think this is specifically for the Java and maven installation steps

File: README.md
Comment (iamitprakash) [reply README.md:51]: > yes @iamitprakash the docker image we are using is a public image.

please answer other parts of my question

File: README.md
Comment (Ajeyakrishna-k) [reply README.md:38]: Can we not have other Java apps in RDS. Should we repeat this everywhere?

Comment (vikhyat187) [reply README.md:38]: We will definetely have, I was telling to Ashif that Ajeykrishna mentioned this for the Java, maven instatllation steps

**Review by vikhyat187 — APPROVED** [review]

**Comment by vikhyat187** [comment]
We should be able to merge this PR after the #76 gets merged, as we have resolved the comment given by Tejas in this PR.

File: README.md
Comment (vikhyat187) [reply README.md:51]: Hi @iamitprakash this we made changes in PR #76

Comment (vikhyat187) [reply README.md:51]: This is merged, so closing this comment

**Review by Ajeyakrishna-k — APPROVED** [review]

**Pull request merged**
